docs/required-trino-apis.md

```diff
@@ -3,12 +3,12 @@ Transport requires applying this [patch](transport-udf-trino.patch) before being
 This patch makes some of the internal UDF classes be visible at the SPI layer. 
 Below we explain why some Transport APIs cannot leverage the APIs offered by the [public SPI UDF model](https://trino.io/docs/current/develop/functions.html).
 
-## [init() method](https://github.com/linkedin/transport/blob/09a89508296a2491f43cc8866d47952c911313ab/transportable-udfs-api/src/main/java/com/linkedin/transport/api/udf/StdUDF.java#L45) is hard to implement on top of Trino-SPI
+## [init() method](https://github.com/linkedin/transport/blob/09a89508296a2491f43cc8866d47952c911313ab/transportable-udfs-api/src/main/java/com/linkedin/transport/api/udf/UDF.java#L45) is hard to implement on top of Trino-SPI
```

Comment:
Could you make sure all the links are still valid?

Reply:
Why do we verify that right now? As I mentioned, we should verify after we change the branch. I am not comfortable if the link will still be good or not after renaming the branch.